model/rules.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+
+
+class RuleType(Enum):
+    COMMIT_MESSAGE_SIZE = "CommitMessageSize"
+
+
+class Rule:
```

Comment:
I think it would be good to have function documentation in these functions from early.

Reply:
Makes sense, should also come in handy for later steps, for documenting how the rules file fields work, and how to implement the rule checker

Reply:
Added document strings on most classes and fields.
Also added README on models package for info on creating new Rules and Scopes as well as how they should work